include/exadg/poisson/spatial_discretization/laplace_operator.cpp

```diff
@@ -54,7 +54,7 @@ LaplaceOperator<dim, Number, n_components>::calculate_penalty_parameter(
   MatrixFree<dim, Number> const & matrix_free,
   unsigned int const              dof_index)
```

Comment:
I think it would be more consistent to also extend this interface by `quad_index`